src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/GetApprenticeshipsResponse.cs

```diff
@@ -17,6 +17,7 @@ public class ApprenticeshipDetailsResponse
             public long Id { get; set; }
             public string FirstName { get; set; }
             public string LastName { get; set; }
+            public string Email { get; set; }
```

Comment:
same with this one?

Reply:
same as above it will be needed in a story or 2

Reply:
no this is fine, but just to let you know, Edit is currently under migration from V1 to V2, may be the story 2 should come after the migration is complete.